DESCRIPTION

```diff
@@ -19,7 +19,7 @@ License: GPL-2
 URL: https://github.com/esqLABS/esqlabsR
 BugReports: https://github.com/esqLABS/esqlabsR/issues
 Depends:
-    ospsuite (>= 12.0.0),
+    ospsuite (>= 11.2.0.9000),
```

Comment:
I think it is ok to use ospsuite 11.2.0 but we don't use the .9000 for ospsuiteR package and so should be removed I think.